base/show.jl

```diff
@@ -132,6 +135,48 @@ function show(io::IO, ::MIME"text/plain", t::AbstractDict{K,V}) where {K,V}
     end
 end
 
+function summary(io::IO, t::AbstractSet)
+    n = length(t)
+    showarg(io, t, true)
+    print(io, " with ", n, (n==1 ? " entry" : " entries"))
```

Comment:
I know dicts use "entries" but I think "elements" makes more sense for sets.

Reply:
Good point, I copy-pasted from dicts' method without thinking twice.